version/prop.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package version
```

Comment:
pkg docs? :)

Reply:
It already has package docs in the other files (that are `!build redo` and `+build !redo`). This is the portable file.